nav2_system_tests/src/behaviors/assisted_teleop/assisted_teleop_behavior_tester.hpp

```diff
@@ -46,7 +46,6 @@ class AssistedTeleopBehaviorTester
 {
 public:
   using AssistedTeleop = nav2_msgs::action::AssistedTeleop;
-  // using GoalHandleBackup = rclcpp_action::ClientGoalHandle<BackUp>;
```

Comment:
? backup

Reply:
Copied over from a old file and I forgot to remove it earlier :(